features/routing/route.feature

```diff
@@ -713,7 +713,6 @@ Feature: Testing route
     """
     When I open web server via the route
     Then the step should fail
-    And expression should be true> @result[:exitstatus] == -1
```

Comment:
The previous step `the step should fail` has checked the result so this one is redundant and can be removed safely.